### PR TITLE
Change intervals to be part of sample specific input

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
In sarek we use these modules with multiple intervals per sample. So to make sure that all samples are run with all intervals, I propose to change the interval input to go along with the sample. 

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

**Review by GCJMackenzie — COMMENTED** [review]
Apologies if I have jumped the gun on this. But hopefully these help speed up getting the tests working :) unsure why genotypegvcfs isn't working though

File: tests/modules/gatk4/genotypegvcfs/main.nf [hunk 1]
```diff
@@ -129,16 +134,14 @@ workflow test_gatk4_genotypegvcfs_gendb_input_intervals {
...
 
-    intervalsBed = file(params.test_data['homo_sapiens']['genome']['genome_bed'], checkIfExists: true)
-
     test_genomicsdb = file(params.test_data['homo_sapiens']['illumina']['test_genomicsdb_tar_gz'], checkIfExists: true)
 
     UNTAR ( test_genomicsdb )
     gendb = UNTAR.out.untar.collect()
     gendb.add([])
-    input = Channel.of([ id:'test' ]).combine(gendb)
+    input = Channel.of([ id:'test' ]).combine(gendb).add(file(params.test_data['homo_sapiens']['genome']['genome_bed'], checkIfExists: true))
```
Comment (GCJMackenzie) [inline tests/modules/gatk4/genotypegvcfs/main.nf:142]: Code doesn't seem to like when .combine() and .add() are mixed. might need to do gendb.add([]) and then gendb.add([path/to/genome_bed]) before performing the combine

File: tests/modules/gatk4/genotypegvcfs/main.nf [hunk 2]
```diff
@@ -97,9 +102,9 @@ workflow test_gatk4_genotypegvcfs_gendb_input {
     UNTAR ( test_genomicsdb )
     gendb = UNTAR.out.untar.collect()
     gendb.add([])
```
Comment (GCJMackenzie) [inline tests/modules/gatk4/genotypegvcfs/main.nf:104]:
```suggestion
    gendb.add([])
    gendb.add([])
```

File: tests/modules/gatk4/genotypegvcfs/main.nf [hunk 3]
```diff
@@ -97,9 +102,9 @@ workflow test_gatk4_genotypegvcfs_gendb_input {
...
-    input = Channel.of([ id:'test' ]).combine(gendb)
+    input = Channel.of([ id:'test' ]).combine(gendb).add([])
```
Comment (GCJMackenzie) [inline tests/modules/gatk4/genotypegvcfs/main.nf:105]:
```suggestion
    input = Channel.of([ id:'test' ]).combine(gendb)
```

File: tests/modules/gatk4/genotypegvcfs/main.nf [hunk 4]
```diff
@@ -117,9 +122,9 @@ workflow test_gatk4_genotypegvcfs_gendb_input_dbsnp {
...
```
Comment (GCJMackenzie) [inline tests/modules/gatk4/genotypegvcfs/main.nf:125]:
```suggestion
    input = Channel.of([ id:'test' ]).combine(gendb)
```

File: tests/modules/gatk4/genotypegvcfs/main.nf [hunk 5]
```diff
@@ -117,9 +122,9 @@ workflow test_gatk4_genotypegvcfs_gendb_input_dbsnp {
     UNTAR ( test_genomicsdb )
     gendb = UNTAR.out.untar.collect()
     gendb.add([])
```
Comment (GCJMackenzie) [inline tests/modules/gatk4/genotypegvcfs/main.nf:124]:
```suggestion
    gendb.add([])
    gendb.add([])
```

**Review by GCJMackenzie — COMMENTED** [review]
These worked locally

File: tests/modules/gatk4/genotypegvcfs/main.nf (hunk 5 above)
Comment (FriederikeHanssen) [reply tests/modules/gatk4/genotypegvcfs/main.nf:126]: thank you! This worked :)

**Review by ggabernet — APPROVED** [review]
Looks good to me!

**Pull request merged**